art/attacks/evasion/projected_gradient_descent/projected_gradient_descent_tensorflow_v2.py

```diff
@@ -342,27 +341,36 @@ def _compute_perturbation(  # pylint: disable=W0221
         if mask is not None:
             grad = tf.where(mask == 0.0, 0.0, grad)
 
-        # Add momentum
+        # Compute gradient momentum
         if decay is not None and momentum is not None:
-            ind = tuple(range(1, len(x.shape)))
-            grad = tf.divide(grad, (tf.math.reduce_sum(tf.abs(grad), axis=ind, keepdims=True) + tol))
-            grad = self.decay * momentum + grad
-            # Accumulate the gradient for the next iter
-            momentum += grad
+            raise NotImplementedError(
+                "Momentum Iterative Attack currently not working with the Tensorflow framework. See issue #2439"
+            )
```

Comment:
Hi @eliegoudout This exception causes the Pylint error W0101: Unreachable code in the Style Check workflow. I think we can remove the exception because you have opened a separate issue which we will fix for the same release as this pull request.

Reply:
So you mean remove "NotImplementedError" line or adding a pylint ignore flag? I think the second approach is much safer personally.

Reply:
I disabled the warning locally as I think it is the safest option atm ([0ddafcd](https://github.com/Trusted-AI/adversarial-robustness-toolbox/pull/2382/commits/0ddafcd83feb70c5a4a91282a7cef4e64e380e91))